tfhe/c_api_tests/CMakeLists.txt

```diff
@@ -33,6 +35,16 @@ file(GLOB TEST_CASES test_*.c)
 foreach (testsourcefile ${TEST_CASES})
     get_filename_component(testname ${testsourcefile} NAME_WLE)
     get_filename_component(groupname ${testsourcefile} DIRECTORY)
+
+    if(${testsourcefile} MATCHES  "test_high_level_threading.c$")
+        # This test requires the <threads.h> header which may not be supported
+        CHECK_INCLUDE_FILE("threads.h" SUPPORTS_C11_THREADS)
+        if (NOT SUPPORTS_C11_THREADS)
+            message(STATUS "skipping ${testsourcefile}: No <threads.h>")
```

Comment:
Apple does not support C11 threading ? even through some homebrew installations ?

Reply:
Does not seems possible, I tried clang and gcc